.travis.yml

```diff
@@ -127,21 +128,18 @@ jobs:
         # Use default port
         - sudo sed -i 's/port = 5433/port = 5432/' /etc/postgresql/10/main/postgresql.conf
         # Use 9.6 auth config:
-        - sudo cp /etc/postgresql/{9.6,10}/main/pg_hba.conf
+        - sudo cp /etc/postgresql/{9.5,10}/main/pg_hba.conf
         - sudo service postgresql restart
         - *default-before-script
 #### PHPUnit tests ###########################
     - &phpunit-tests
-      php: 5.6
```

Comment:
We still support debian jessie with PHP 5.6, so we should not drop the test here

Reply:
Missed that point. Thanks for pointing it out.